test/jdk/com/sun/net/httpserver/bugs/B6968351.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2005, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 6968351
+ * @summary  tcp no delay not required for small payloads
+ * @library /test/lib
+ * @run main/timeout=5 B6968351
+ * @run main/othervm -Dsun.net.httpserver.nodelay=false B6968351
```

Comment:
```suggestion
 * @run main/othervm/timeout=5 -Dsun.net.httpserver.nodelay=false B6968351
```
you need the timeout on every `run` line.

The test will be run as many times as there are `run` lines; I'm assuming you actually want to run it twice here, correct?

Reply:
No I didn't. TBH, this is my first usage of jtreg. I was duplicating other tests somewhat blindly. I will fix.